src/Core/Entity/Image.php

```diff
@@ -11,10 +12,13 @@
  */
 class Image
 {
+    /** Bin path */
+    const IM_IDENTITY_COMMAND = '/usr/bin/identify';
```

Comment:
a Image class should not have a execute function, it should pass through the image processor.